data_steward/analytics/cdr_ops/run_rdr_qc_report.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+import papermill as pm
+import argparse
+import logging
+
+
+NOTEBOOK_TEMPLATE_NAME = "rdr_qc_template.ipynb"
+EXECUTED_NOTEBOOK_NAME = "rdr_qc_export.ipynb"
+
+logging.basicConfig(
+    level = logging.INFO,
+    format = '[%(levelname)s] %(asctime)s - %(message)s',
+    handlers=[logging.StreamHandler()]
+)
+logger = logging.getLogger(__name__)
```

Comment:
We should use `pipeline_logging.configure` and relocate this to the main function similar to [here](https://github.com/all-of-us/curation/blob/develop/data_steward/retraction/retract_deactivated_pids.py#L588).